vignettes/using-board-url.Rmd

```diff
@@ -0,0 +1,208 @@
+
+```{r}
+#| message: false
+web_board %>% pin_read("mtcars") %>% head()
+```
+
+We can also read the first version:
+
+```{r}
+#| message: false
+web_board %>% pin_read("mtcars", version = versions$version[[1]]) %>% head()
+```
+
+The next section shows some ways you can integrate web-based boards into your workflows.
+
+## Publishing platforms
+
+The goal of this section is to illustrate, for a few platforms, the particular way to publish a board as a part of a website.
+
+### Quarto
+
+Using Quarto, you can [publish](https://quarto.org/docs/publishing/) to GitHub Pages, Quarto Pub, and Netlify, among others.
+
+Let's say you set up a board using a directory called `"pins-board"`; of course, you may wish to use a different name.
+Here's how to set up your Quarto project:
+
+1. In your code, create a `board_folder("pins-board")` in the root directory:
+  - Write to the board using `pin_write()`.
+  - At the end, call `write_board_manifest()`.
+
+2. Add the board folder as a [Quarto site-resource](https://quarto.org/docs/websites/website-tools.html#site-resources):
+
+  - At the top level of your `_quarto.yml` file:
+  
+    ```yml
+    resources:
+      - pins-board
+    ```
+
+  - You may wish to add `/pins-board/` to your project's `.gitignore` file.
+
+Here's an example of a [Quarto repository](https://github.com/ijlyttle/quarto-pins) and its [published website](https://ijlyttle.github.io/quarto-pins/).
+
+### pkgdown
+
+Pins offers another way for R-package developers to share data associated with a package. 
+Publishing a package dataset as a pin can let you extend your data's "audience" to Python users.
+
+Using [pkgdown](https://pkgdown.r-lib.org/articles/customise.html#additional-html-and-files), any files you save in the directory `pkgdown/assets/` will be copied to the website's root directory when `pkgdown::build_site()` is run.
+
+The [R Packages book](https://r-pkgs.org/data.html#sec-data-data-raw) suggests using a folder called `data-raw` for working with datasets; this can be adapted to use pins. You would start with `usethis::use_data_raw()`.
+
+In your data-raw file, you would wrangle and clean your datasets, then `usethis::use_data()` to create the datasets for your package.
+To offer such datasets on a web-based board, in your data-raw file you would:
+
+  - Create a board: `board_folder(here::here("pkgdown/assets/pins-board"))`; you might use a different name than `"pins-board"`.
+  - Write your datasets to the board using `pin_write()`.
+  - At the end, call `write_board_manifest()`.
+
+The R Packages book offers this observation on CRAN and package data:
+
+> Generally, package data should be smaller than a megabyte - if it’s larger you’ll need to argue for an exemption.
+
+Publishing a board on your pkgdown site provides a way to offer extended versions of your data.
+
+Here's an example of a [toy package](https://github.com/ijlyttle/pkgpins) and its [pkgdown site](https://ijlyttle.github.io/pkgpins/).
```

Comment:
I think I'm going to take this out of the vignette since we're not going to share a similar example for S3. However @ijlyttle would you be OK with keeping that repo + site up at least until we publish the next pins release blog post (plus a while after that), so I can link to them from there?

Reply:
No worries - I'll keep the repo & site up